test/sanity/collections/issue1771EqualEqual.wtest

```diff
@@ -0,0 +1,14 @@
+/**
+  * Scenario 1: A custom class overriding ==
+	*/
+class SpecialClass {
+	override method ==(other) = true
+}
+
+/**
+  * Tests
+	*/
+test "issue 1771 - defining a set for a custom class overriding (==)" {
```

Comment:
Not a great name for the file/test

Reply:
They are already in `collection` folder, and I'd like to maintain short scenarios (I thought you also preferred that)